source/includes/_opportunities.md

```diff
@@ -0,0 +1,278 @@
+            "opportunity_contact_roles": [],
+            "salesforce_record_url": "https://na85.salesforce.com/0061U000XXXXXXXXXX",
+            "account": {
+                "id": "5c14223d945746a058358f8f",
+                "domain": "apollooportunity.com",
+                "name": "Apollo Oportunity Limited",
+                "team_id": "5c10XXXXXXXXXXXXXXXXXXXX",
+                "typed_custom_fields": {},
+                "organization_id": null,
+                "account_stage_id": "5c100XXXXXXXXXXXXXXXXXXXX",
+                "source": "salesforce",
+                "original_source": "salesforce",
+                "owner_id": "5c10XXXXXXXXXXXXXXXXXXXX",
+                "created_at": "2018-12-14T21:35:57.141Z",
+                "phone": "(512) 757-6000",
+                "phone_status": "no_status",
+                "test_predictive_score": null,
+                "hubspot_id": null,
+                "salesforce_id": "0011U000XXXXXXXXXX",
+                "crm_owner_id": "0051U000XXXXXXXXXX",
+                "parent_account_id": null,
+                "sanitized_phone": "+1512XXXXXXX",
+                "account_playbook_statuses": [],
+                "existence_level": "full",
+                "label_ids": [],
+                "modality": "account",
+                "salesforce_record_url": "https://na85.salesforce.com/0011U000XXXXXXXXXX",
+                "contact_emailer_campaign_ids": [],
+                "contact_campaign_status_tally": {},
+                "num_contacts": 0,
+                "last_activity_date": null,
+                "persona_counts": {}
+            }
+        }
+    ]
+```
+
+This endpoint returns all opportunities you have Created.
+
+`GET https://api.apollo.io/v1/opportunities`
+
+## View Opportunity
+
+> Sample request:
+
+```shell
+curl -X GET -H "Content-Type: application/json" -H "Cache-Control: no-cache" "https://api.apollo.io/v1/opportunities/REPLACE_WITH_OPPORTUNITY_ID?api_key=YOUR_API_KEY_HERE"
+```
+
+> Sample response:
+
+```json
+{
+  "opportunity": {
+    "id": "5c14XXXXXXXXXXXXXXXXXXXX",
+    "team_id": "5c10XXXXXXXXXXXXXXXXXXXX",
+    "owner_id": "5c10XXXXXXXXXXXXXXXXXXXX",
+    "salesforce_owner_id": "0051U000001FAqXQAW",
+    "amount": 90000.0,
+    "closed_date": "2018-08-27T15:00:00.000+00:00",
+    "account_id": "5c14XXXXXXXXXXXXXXXXXXXX",
+    "description": null,
+    "is_closed": true,
+    "is_won": true,
+    "name": "Grand Apollo SLA",
+    "stage_name": "Closed Won",
+    "opportunity_stage_id": "5c10XXXXXXXXXXXXXXXXXXXX",
+    "source": "crm",
+    "salesforce_id": "0061U000XXXXXXXXXX",
+    "created_at": "2018-12-14T21:36:02.191Z",
+    "existence_level": "full",
+    "opportunity_contact_roles": [],
+    "salesforce_record_url": "https://na85.salesforce.com/0061U000XXXXXXXXXX",
+    "num_contacts": 0
+  }
+}
+```
+
+This endpoint returns the full information about an opportunity.
+
+`GET https://api.apollo.io/v1/opportunities/REPLACE_WITH_OPPORTUNITY_ID`
```

Comment:
Providing an example of `REPLACE_WITH_OPPORTUNITY_ID` would be good. Similar to [this](https://github.com/apolloio/apollo-api-docs/pull/13/files#diff-6f010c6b732b5c0e0e87c8c3025000847791c8b7960ea16c7a79a513a4a943c1R52).